src/Components/Todo/TodoItem.js

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import { connect, useDispatch } from "react-redux";
+import { Button, Grid, Typography } from "@material-ui/core";
+import PropTypes from "prop-types";
+import { deleteTodo, toggleTodo } from "../../redux/actions";
+import myStyles from "../../MaterialUI/myStyles";
+
+const TodoItem = (props) => {
+  const { state } = props;
+  const tasks = state.storage.find((task) => task.list === state.currentList)
+    .taskStore;
+  const dispatch = useDispatch();
+  const classes = myStyles();
+
+  return (
+    <Grid>
+      {tasks.map((todo) => (
+        <Grid
+          key={todo.task}
+          className={classes.itemsSeparation}
+          justify="center"
+          alignItems="baseline"
+          container
+          item
+          sm={12}
+        >
+          <Grid item sm={9}>
+            <Typography
+              variant="body1"
+              className={`${todo.completed && classes.completedTask}`}
+            >
+              {todo.task}
+            </Typography>
+          </Grid>
+
+          <Grid item sm={1}>
+            <Button
+              onClick={() => {
```

Comment:
You can remove brackets here and leave just:

`() => dispatch(toggleTodo(todo.task));`